node/podcontroller.go

```diff
@@ -69,7 +69,13 @@ type PodLifecycleHandler interface {
 // of pod status updates asynchronously.
 type PodNotifier interface {
 	// NotifyPods instructs the notifier to call the passed in function when
-	// the pod status changes.
+	// the pod status changes. It should be called pod mutated with the new
```

Comment:
I can't follow what this means. Maybe you ate some words or something?

Reply:
```
It should be called when a pod's status changes.
```